tests/platform_tests/api/test_sfp.py

```diff
@@ -668,17 +671,48 @@ def test_reset(self, request, duthosts, enum_rand_one_per_hwsku_hostname, localh
         # TODO: Verify that the transceiver was actually reset
         duthost = duthosts[enum_rand_one_per_hwsku_hostname]
         skip_release_for_platform(duthost, ["202012"], ["arista", "mlnx"])
+        port_index_to_info_dict = {}
 
         for i in self.sfp_setup["sfp_test_port_indices"]:
             info_dict = sfp.get_transceiver_info(platform_api_conn, i)
             if not self.expect(info_dict is not None, "Unable to retrieve transceiver {} info".format(i)):
                 continue
+            port_index_to_info_dict[i] = info_dict
 
             ret = sfp.reset(platform_api_conn, i)
             if self.is_xcvr_resettable(request, info_dict):
                 self.expect(ret is True, "Failed to reset transceiver {}".format(i))
             else:
                 self.expect(ret is False, "Resetting transceiver {} succeeded but should have failed".format(i))
+
```

Comment:
@sdszhang Please add a 5s delay (I2C_WAIT_TIME_AFTER_RESET) here to allow the I2C interface to recover post sfp reset.